RELEASE-NOTES.md

```diff
@@ -1,3 +1,59 @@
+# Riak CS 1.5.4 Release Notes
+
+## Bugs Fixed
+
+- Disable previous riak object after backpressure sleep is triggered
+  [riak_cs/#1041](https://github.com/basho/riak_cs/pull/1041). This
+  change prevents unnecessary siblings increase when backpressure is
+  triggered under high upload concurrency, and uploads are interleaved
+  during the backpressure sleep.
+- Fix wrong path rewrite in S3 API by unnecessary unquote
+  [riak_cs/#1040](https://github.com/basho/riak_cs/pull/1040). For
+  example, in case when '+' is included in path, '+' (plus, `%2B` in
+  quoted representation) in object key has been mistakenly unquoted to
+  become ' ' (whitespace, `%20` in quoted representaiton). This fix
+  also addresses
+  [riak_cs/#910](https://github.com/basho/riak_cs/pull/910) and
+  [riak_cs/#977](https://github.com/basho/riak_cs/pull/977).
+
```

Comment:
For readers who don't want to refer pull requests and related issues, I want to add here another kind of effected character sequence `%[0-9a-fA-F][0-9a-fA-F]`.